gcpdiag/lint/gcf/warn_2021_001_cloudfunctions_deprecated_runtimes.py

```diff
@@ -1,7 +1,7 @@
 # Lint as: python3
 """Cloud Functions don't use deprecated runtimes.
 
-Nodejs8 and Go111 runtimes are deprecated.
+Nodejs8, Nodejs6 and Go111 runtimes are deprecated.
```

Comment:
nit: maybe sort? "Go111, Nodejs6, and Nodejs8" (note also the comma before the "and"). Or should this be replaced with a list like: the following runtimes are deprecated: list.

Reply:
Done